frontend/app/ui/subscriptions/list/template.hbs

```diff
@@ -92,5 +92,19 @@
       </table>
     </div>
 
+    <p class="subscription-export">
+      <button
+        {{on "click" this.export}}
+         class="subscription-export__button"
```

Comment:
Indentation is off here

Reply:
Interesting! `block-indentation` seems to have missed that. I'll correct it.

Reply:
Done